README.md

```diff
@@ -1,149 +1,118 @@
-
-- **Generated libraries produced by a managed compiler**: Language-specific stub generation using Buf’s high-performance, drop-in protoc replacement.
-
-Over time, our goal is to make the BSR the only tool you need to manage your Protobuf workflow from end to end. To that end, there's a lot we are planning with the Buf Schema Registry. For a quick overview, see our [roadmap](https://docs.buf.build/roadmap/).
-
-## Where to go from here
-
-To install Buf, proceed to [installation](https://docs.buf.build/installation/). This includes links to an example
-repository for Travis CI and GitHub Actions integration.
-
-Next, we recommend completing the [tour](https://docs.buf.build/tour/introduction/). This tour should only take about 10 minutes, and
-will give you an overview of most of the existing functionality of Buf.
-
-After completing the tour, check out the remainder of the docs for your specific areas of interest.
-We've aimed to provide as much documentation as we can for the various components of Buf to give
-you a full understanding of Buf's surface area.
+# Buf
 
-Finally, [follow the project on GitHub](https://github.com/bufbuild/buf),
-and [contact us](https://docs.buf.build/contact/) if you'd like to get involved.
+[![CI](https://github.com/bufbuild/buf/workflows/ci/badge.svg)][badges.ci]
+[![Release](https://img.shields.io/github/v/release/bufbuild/buf?include_prereleases)][badges.release]
+[![Homebrew](https://img.shields.io/badge/homebrew-v1.0.0--rc12-blue)][badges.homebrew]
+[![AUR](https://img.shields.io/aur/version/buf)][badges.aur]
+[![Slack](https://img.shields.io/badge/slack-buf-%23e01563)][badges.slack]
+[![Twitter](https://img.shields.io/twitter/follow/bufbuild?style=social)][badges.twitter]
+[![Docker](https://img.shields.io/docker/pulls/bufbuild/buf)][badges.docker]
+[![License](https://img.shields.io/github/license/bufbuild/buf?color=blue)][badges.license]
+
+The [`buf`][buf] CLI is a tool for working with [Protocol Buffers][protobuf] APIs, offering a range of features not found in the standard `protoc` compiler, including:
+
+- The ability to manage Protobuf assets, including [plugins] and [templates], on the [Buf Schema Registry][bsr] (BSR).
+- A [linter][lint_usage] that enforces good API design choices and structure.
+- A [breaking change detector][breaking_usage] that enforces compatibility at the source code or wire level.
+- A [generator][generate_usage] that invokes your protoc plugins based on a configurable [template][templates].
+  A [protoc replacement][protoc] that uses Buf's [high-performance Protobuf compiler][compiler].
+- A configurable file [builder][build_usage] that produces [Images], our extension of Protobuf's native [FileDescriptorSets][filedescriptorset].
+
+See more [features] below.
+
+## Installation
+
+### Homebrew
+
+You can install `buf` using [Homebrew][brew] (macOS or Linux) through the [`bufbuild/bufbrew`][tap] tap.
+
+```sh
+brew tap bufbuild/bufbrew install buf
```

Comment:
What is the `bufbuild/bufbrew` tap? I've never heard of this before - you can just do `brew install bufbuild/buf/buf`